sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -1485,15 +1485,19 @@ def __init__(self, **kwargs):
         self.string_index_type = kwargs.get("string_index_type", "UnicodeCodePoint")
 
     def __repr__(self, **kwargs):
-        return "RecognizePiiEntitiesAction(model_version={}, domain_filter={}, string_index_type={})" \
-            .format(self.model_version, self.domain_filter, self.string_index_type)[:1024]
+        return "RecognizePiiEntitiesAction(model_version={}, domain_filter={}, string_index_type={}".format(
+            self.model_version,
+            self.domain_filter,
+            self.string_index_type
+        )[:1024]
 
     def to_generated(self):
         return _latest_preview_models.PiiTask(
             parameters=_latest_preview_models.PiiTaskParameters(
                 model_version=self.model_version,
                 domain=self.domain_filter,
-                string_index_type=self.string_index_type
+                string_index_type=self.string_index_type,
+                pii_entity_categories=self.categories_filter
```

Comment:
still need to remove `pii_entity_categories=self.categories_filter` from `to_generated`

Reply:
Removed!